### ISSUE TITLE
document how to define a schema with the Go client

### ISSUE THREAD

**Issue by dalu** [dgraph-io/dgraph#1717]
As written in the forum,

It is not clear how to create a schema with the Go client.
That is true for 0.8 and for 0.9
The documentation is at https://docs.dgraph.io/query-language/#schema

SchemaUpdate
https://godoc.org/github.com/dgraph-io/dgraph/protos#SchemaUpdate
is used in

Dgraph.CheckSchema
https://godoc.org/github.com/dgraph-io/dgraph/client#Dgraph.CheckSchema

What does CheckSchema mean? Is that how you define a schema?
If so what does a schemadefinition look like in this context?
Let's take this schema definition from the documentation
```
mutation {
  schema {
    name: string @index(exact, fulltext) @count .
    age: int @index(int) .
    friend: uid @count .
    dob: dateTime .
    location: geo @index(geo) .
    occupations: [string] @index(term) .
  }
}
```

For this example how would one define a schema using the Go rpc or http client?

**Comment by manishrjain** [comment]
Agreed. The current documentation is lacking w.r.t. schema. We'll fix it before v0.9 release.

**Comment by pawanrawal** [comment]
Here is an example of how to do it using the latest version.
https://docs.dgraph.io/clients/#alter-the-database
